patroni/dcs/__init__.py

```diff
@@ -458,9 +490,20 @@ def cancel_initialization(self):
         """ Removes the initialize key for a cluster """
 
     @abc.abstractmethod
-    def delete_cluster(self):
+    def delete_cluster(self, index=None):
```

Comment:
Whoops. That explains the codacy bot complaining about interface mismatch... Will fix.

On that note, should I rebase this series to something cleaner when things start to stabilize?